pkgs/by-name/do/dokieli/package.nix

```
@@ -52,7 +53,12 @@ mkYarnPackage rec {
     homepage = "https://github.com/linkeddata/dokieli";
     license = lib.licenses.mit;
     platforms = lib.platforms.all;
-    maintainers = with lib.maintainers; [ shogo ];
+    maintainers =
+      with lib.maintainers;
+      [
+        shogo
+      ]
+      ++ lib.teams.ngi.members;
```

No need to make the diff bigger than what it needs to be:

```suggestion
    maintainers = with lib.maintainers; [ shogo ] ++ lib.teams.ngi.members;
```

---

@eljamm I have made these correction. but some checks are still failing. 
please could you have a look?